test/benchmark/sysbench-cpu/config.json

```diff
@@ -1,5 +1,7 @@
 {
     "alert_threshold": "130%",
     "pattern": "avg:",
-    "field": "NF"
+    "field": "NF",
```

Comment:
I am curious about the meaning of "NF".

Reply:
NF is a built-in variable of `awk`, indicating the number of fields in the current line. It means the last field here.